client/src/index.js

```diff
@@ -1,28 +1,30 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import OverviewApp from './productOverview/overviewApp.jsx';
 
 
-class App extends React.Component {
```

Comment:
I assumed this app class is where we would collaborate on rendering all of our components together.